### PR TITLE
Add swagger config

### PR DESCRIPTION
**Pull request by s-kakol**
Basic swagger configuration.

Example:
swagger

### CODE REVIEW

File: src/utils/swagger.config.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {
+  DocumentBuilder,
+  OpenAPIObject,
+  SwaggerCustomOptions,
+} from '@nestjs/swagger';
+
+export const swaggerConfig = (): Omit<OpenAPIObject, 'paths'> => {
+  return new DocumentBuilder()
+    .setTitle('Blood Donor API')
+    .setDescription('API documentation for the Blood Donor Mobile App')
```
Comment (adjurk) [inline src/utils/swagger.config.ts]: It's actually the API docs for the backend API, which is used by the mobile app ;)

File: src/utils/swagger.config.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    .setVersion('0.1')
```
Comment (adjurk) [inline src/utils/swagger.config.ts]: Let's make it `0.1.0` for semantic versioning 👍

**Review by adjurk — CHANGES_REQUESTED** [review]
Two changes and it's ready!

File: src/utils/swagger.config.ts (hunk 1 above)
Comment (s-kakol) [reply src/utils/swagger.config.ts]: Done

File: src/utils/swagger.config.ts (hunk 2 above)
Comment (s-kakol) [reply src/utils/swagger.config.ts]: True, done.

**Review by matgrube — APPROVED** [review]
LGTM!

**Review by adjurk — APPROVED** [review]

**Pull request merged**